src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -528,3 +540,9 @@ NavCategoryGroup ^ NavCategoryGroup::CreateConverterCategory()
     return ref new NavCategoryGroup(
         NavCategoryGroupInitializer{ CategoryGroupType::Converter, L"ConverterModeTextCaps", L"ConverterModeText", L"ConverterModePluralText" });
 }
+
+void NavCategory::Dummy()
```

Comment:
What is the Dummy() used to do?

Reply:
It is a stub that was used to test a crashing issue.
This is a great finding. I've removed this crap in the latest commit.